test/pagespecs/PaymentSummaryPageSpec.scala

```diff
@@ -59,6 +59,6 @@ class PaymentSummaryPageSpec extends ItSpec {
     paymentSummaryPage.assertPageIsDisplayed
 
     paymentSummaryPage.clickContinue()
-    monthlyPaymentAmountPage.assertPageIsDisplayedAltPath(-100: Int)
+    monthlyPaymentAmountPage.assertPageIsDisplayedAltPath(400, 2390)
```

Comment:
pedantic: numbers can't be understood without looking at definition  of `assertPageIsDisplayedAltPath`

Reply:
yes, any clue how to change it?